resources/kyverno/admission-control-policy/kube.cue

```diff
@@ -0,0 +1,99 @@
+package kube
```

Comment:
I wonder if the package should be something like "kyverno" or "kyvernoAdmissionControlPolicy"?

Reply:
I agree. I made it too generic I will change that. I think `kyverno` should suffice.